esp/smc/SMCLib/InfoCacheReader.cpp

```diff
@@ -0,0 +1,77 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2019 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+#pragma warning (disable : 4786)
```

Comment:
The code is copied from ws_smcService.cpp to disable VC warning c4786. Should I remove it?

Reply:
I would not remove it unless you're sure it is no longer relevant.
But it would be good to know why it's there (pity it doesn't have comment).
Perhaps you can see where/why it added via git blame.
And/or try removing it and see if you get a compiler warning.

Reply:
The code has been there since we moved the HPCC from svn to git. It may not cause a warning in Linux build, but, not sure in Windows build (I cannot do Windows build now).